README.md

```diff
@@ -1,6 +1,6 @@
 # CIDR Matcher
 
-Fast CIDR matcher. Given an input IPv4 address, it checks if it's inside a set of IP ranges, expressed in CIDR notation. This module is based upon the great [node-ip](https://github.com/indutny/node-ip.git) module.
+Fast CIDR matcher. Given an input IPv4 or IPv6 address, it checks if it's inside a set of IP ranges, expressed in CIDR notation. This module is based upon the great [node-ip](https://github.com/indutny/node-ip.git) module.
```

Comment:
I believe is still referencing the old `ip` module. We should now mention `ip6addr` and link to https://github.com/joyent/node-ip6addr

Reply:
Thanks for noticing it! Fixed